web/client/plugins/Widgets.jsx

```diff
@@ -60,10 +81,16 @@ compose(
     compose(
         heightProvider({ debounceTime: 20, closest: true, querySelector: '.fill' }),
         widthProvider({ overrideWidthProvider: false }),
-        withProps(({width, height, maximized, mapLayout} = {}) => {
+        withProps(({ isMobileAgent }) => {
```

Comment:
please try to apply this patch:

- review of width/height breakpoints
- move the singleWidgetLayoutBreakpoint to a prop

[resize.zip](https://github.com/geosolutions-it/MapStore2/files/9703862/resize.zip)

Reply:
thanks, applied the patch